### ISSUE TITLE
ViaVersion kicking players out

### ISSUE THREAD

**Issue by AccioAce** [ViaVersion/ViaVersion#3576]
### `/viaversion dump` Output

https://dump.viaversion.com/841eb5d08b9975ab5bf8bf79749ac076096e487a1c0e4149e72775066e56d205

### Console Error

```
I did not detect console error on my end.
```

### Bug Description

I am using latest viaversion and viabackward - so I have it updated to latest one. I notice players using **1.20.3 and 1.20.4** while at it if someone speaks in discord that goes through in-game those users that using 1.20.3 gets kick from the server. Here is the error https://prnt.sc/tumw5jXWNLDZ

**Scarsz -- DiscordSRV dev said it isn't his plugin and I use report this to Viaversion Developer.**

### Steps to Reproduce

If anyone players using version **1.20.3 and 1.20.4** while using talking from discord to send to in-game through **DicsordSRV** setup it kicks them out with the error.

### Expected Behavior

Expected to not kick players out when they are using DiscordSRV minechat.

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried the latest build(s) from https://ci.viaversion.com/ and the issue still persists.

**Comment by Barvalg** [comment]
![Platform: velocity%203.2.0--SNAPSHOT%20%28git--37fedf50--b260%29](https://img.shields.io/badge/Platform-velocity%203.2.0--SNAPSHOT%20%28git--37fedf50--b260%29-3498db?style=flat-square) </br>![Proxy is 41 builds behind](https://img.shields.io/badge/Your%20proxy%20is%2041%20build(s)%20behind.%20Please%20update%20to%20the%20latest%20version-orange?style=flat-square) </br>![ViaVersion (4.9.3-SNAPSHOT): Even with master](https://img.shields.io/badge/ViaVersion%20(4.9.3--SNAPSHOT)-Even%20with%20master-brightgreen?style=flat-square) </br>![ViaBackwards(4.9.1): 7 commits behind master](https://img.shields.io/badge/ViaBackwards%20(4.9.1)-7%20commit(s)%20behind%20master-yellow?style=flat-square) </br><br><h3>Please try the **latest** build(s) from [https://ci.viaversion.com/](https://ci.viaversion.com/). In case the issue still persists send the new dump</h3>

**Comment by mercurialmusic** [comment]
Similar thing still happening to me as reported after #3568 was closed.

**Comment by Kichura** [comment]
Can you update your proxy and viabackwards to latest builds to see if problems persist?

**Comment by AccioAce** [comment]
> Can you update your proxy and viabackwards to latest builds to see if problems persist?

I've tried with latest build
of viaversion and viabackward no luck it seems.

I hope this can be fixed and resolved very soon.

**Comment by Kichura** [comment]
> > Can you update your proxy and viabackwards to latest builds to see if problems persist?
> 
> I've tried with latest build of viaversion and viabackward no luck it seems.
> 
> I hope this can be fixed and resolved very soon.

and what happens if you attempted to run via* on the spigot/paper servers instead, does it make any difference?

**Comment by AccioAce** [comment]
> > > Can you update your proxy and viabackwards to latest builds to see if problems persist?
> > 
> > 
> > I've tried with latest build of viaversion and viabackward no luck it seems.
> > I hope this can be fixed and resolved very soon.
> 
> and what happens if you attempted to run via* on the spigot/paper servers instead, does it make any difference? _(If yes; then something is preventing viaversion from working properly on velocity)_

I am not to sure but I can set it up and test it to see. As most of my productions run on a network.

**Comment by Jo0001** [comment]
@AccioAce is this still an issue or is it fixed in the meantime?

**Comment by vspcoderz** [comment]
Same Error Is Getting Me With 1.21.4 Also It kicking Out GeyserMC Player If I remove Viaversion All Working Fine

**Comment by Jo0001** [comment]
Please open a new issue

**Comment by kennytv** [comment]
Make sure you are actually using the latest Hangar/modrinth/ci builds and not some self-compiled and still cached or few-days old build, even if the commit hash matches

**Comment by vspcoderz** [comment]
https://github.com/ViaVersion/ViaVersion/issues/3576#issuecomment-2966475528

i am using monrith build and github build alsi
